spec/rake/rake_tasks_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+require 'rake'
+Rails.application.load_tasks
+
+describe 'send_mail namespace rake tasks' do
+  describe 'send_mail:send_statistic_to_super_admin' do
+    Rake::Task.task_defined?("send_mail:send_statistic_to_super_admin")
```

Comment:
this thing really does nothing
1. no example definition
2. test fails only if error is raised within an example, not a context definition